firebase-messaging/src/main/java/com/google/firebase/messaging/TopicOperation.java

```diff
@@ -117,6 +119,7 @@ public int hashCode() {
   }
 
   @StringDef({TopicOperations.OPERATION_SUBSCRIBE, TopicOperations.OPERATION_UNSUBSCRIBE})
+  @Retention(RetentionPolicy.SOURCE)
   @interface TopicOperations {
     String OPERATION_SUBSCRIBE = "S";
     String OPERATION_UNSUBSCRIBE = "U";
```

Comment:
Will these two values be discarded at compile time if we add “SOURCE” Retention policy?
If so, we might need to pull these two string out.